server/src/main/java/org/candlepin/audit/EventFactory.java

```diff
@@ -210,14 +210,23 @@ public Event subscriptionExpired(Subscription subscription) {
 
     public Event complianceCreated(Consumer consumer,
         Set<Entitlement> entitlements, ComplianceStatus compliance) {
+        StringBuilder eventDataBuilder = new StringBuilder()
+            .append("{\"consumer\": ")
+            .append("{\"uuid\": \"")
```

Comment:
Why is this a nested field? It could just use consumer_uuid or some such as the key so users of the eventData don't need to dig further into the tree.

Reply:
I had a doubt about it, but I believe I misunderstood what the document said. I thought the status should be a top-level item, while the uuid not. It does make sense if they're both top-level, so I'll change it.